internal/runtimehandlerhooks/high_performance_hooks.go

```diff
@@ -25,38 +27,116 @@ const (
 
 const (
 	annotationCPULoadBalancing = "cpu-load-balancing.crio.io"
+	annotationCPUQuota         = "cpu-quota.crio.io"
+	annotationIRQLoadBalancing = "irq-load-balancing.crio.io"
+	annotationTrue             = "true"
 	schedDomainDir             = "/proc/sys/kernel/sched_domain"
+	irqSmpAffinityProcFile     = "/proc/irq/default_smp_affinity"
+	cgroupMountPoint           = "/sys/fs/cgroup"
 )
 
 // HighPerformanceHooks used to run additional hooks that will configure a system for the latency sensitive workloads
 type HighPerformanceHooks struct{}
 
 func (h *HighPerformanceHooks) PreStart(ctx context.Context, c *oci.Container, s *sandbox.Sandbox) error {
 	log.Infof(ctx, "Run %q runtime handler pre-start hook for the container %q", HighPerformance, c.ID())
+
+	if isCgroupParentBurstable(s) {
+		log.Infof(ctx, "Container %q is a burstable pod. Skip PreStart.", c.ID())
```

Comment:
Do we wanna use `Debugf` messages here and down below instead?

Reply:
This one is on the boundary - the information is not verbose enough to make it to debug level so I feel like to keep it at the info level.